identity-credential/src/credential/builder.rs

```diff
@@ -234,17 +233,29 @@ mod tests {
 
     assert_eq!(credential.context.len(), 2);
     assert_eq!(credential.context.get(0).unwrap(), Credential::<Object>::base_context());
-    assert_eq!(credential.context.get(1).unwrap(), "https://www.w3.org/2018/credentials/examples/v1");
+    assert_eq!(
```

Comment:
I am curious why these specifically are spread over multiple lines, but others aren't. Is it based on the length?

Reply:
this is just due to `rustfmt`, it was previously disabled so this code wasn't being formatted